app/src/main/java/ch/epfl/sweng/erpa/services/dummy/database/DummyDataService.java

```diff
@@ -0,0 +1,120 @@
+package ch.epfl.sweng.erpa.services.dummy.database;
+
+import android.content.Context;
+import android.util.Log;
+
+import com.annimon.stream.Collectors;
+import com.annimon.stream.Exceptional;
+import com.annimon.stream.Optional;
+import com.annimon.stream.Stream;
+import com.annimon.stream.function.Function;
+
+import org.yaml.snakeyaml.Yaml;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.Arrays;
+import java.util.Set;
+
+import ch.epfl.sweng.erpa.model.UuidObject;
+import ch.epfl.sweng.erpa.services.DataService;
+import lombok.Getter;
+
+import static android.content.ContentValues.TAG;
+
+public abstract class DummyDataService<T extends UuidObject> implements DataService<T> {
+    public final static String SAVED_DATA_FILE_EXTENSION = ".yaml";
```

Comment:
make private, this concerns only the implementation of the service

Reply:
Has to be package private, for the tests to work